util/spawn-matches.js

```diff
@@ -31,10 +35,10 @@ const time = getOrder(now);
 
 const current = season.weeks.find(week => {
   const date = getOrder(new Date(week.date));
-  console.log('week: ', date, week.date);
-  console.log('today:', time);
-  console.log('diff:', date - time);
-  if (date >= time && date - time < 7) {
+  console.log('week: ', week.n, date, week.date);
+  console.log('today:  ', time);
+  console.log('diff:  ', date - time);
+  if (date >= time) {
```

Comment:
It was a small miracle that the script created the correct week before this change. The flawed `getOrder` method just happened to create the diffs needed to satisfy the condition, but that starts to break down across month boundaries (currently Sep -> Oct).